pkg/virt-api/webhooks/arm64.go

```diff
@@ -68,6 +69,37 @@ func verifyInvalidSetting(field *k8sfield.Path, spec *v1.VirtualMachineInstanceS
 			Field:   field.Child("domain", "cpu", "model").String(),
 		}, false
 	}
+	if spec.Domain.Devices.Disks != nil {
+		busList := []string{}
+		for i := range spec.Domain.Devices.Disks {
+			disk := &spec.Domain.Devices.Disks[i].DiskDevice
+			if disk.Disk != nil {
+				busList = append(busList, disk.Disk.Bus)
+			}
+			if disk.CDRom != nil {
+				busList = append(busList, disk.CDRom.Bus)
+			}
+			if disk.LUN != nil {
+				busList = append(busList, disk.LUN.Bus)
+			}
+		}
+		fmt.Println(busList)
+		for _, bus := range busList {
+			switch bus {
+			case "":
+			case "sata":
+			case "virtio":
+			case "scsi":
+				continue
+			default:
+				return metav1.StatusCause{
+					Type:    metav1.CauseTypeFieldValueNotSupported,
+					Message: "Arm64 not support this disk bus type, please use virtio or scsi",
+					Field:   bus,
```

Comment:
The field value typically points to the actual field which is causing the issue. For instance something like:

```
field.Child("domain", "devices", "disks").Index(idx).Child("disk", "bus").String()
```

I think it would work best for you if you would check for each device type immediately if the bus is correct and replace the `busList` with a list of errors, so that you can construct appropriate errors directly when visiting the individual device.